projects/core/src/cart/store/reducers/multi-cart.reducer.ts

```diff
@@ -0,0 +1,52 @@
+import { Cart } from '../../../model/cart.model';
+import { LoaderAction } from '../../../state/utils/loader/loader.action';
+import { CartActions } from '../actions/index';
+
+export const activeCartInitialState = '';
+
+export function activeCartReducer(
+  state = activeCartInitialState,
+  action:
+    | CartActions.CartAction
+    | CartActions.CartEntryAction
```

Comment:
Does this reducer accept cart entry actions?

Reply:
No, removed